src/app/donation-start/donation-start-form/donation-tipping-slider/donation-tipping-slider.component.ts

```diff
@@ -56,8 +57,13 @@ export class DonationTippingSliderComponent implements OnInit, AfterContentInit,
   position: number;
   disableDefaults: boolean = false;
 
+  private readonly viewportChange = this.viewPortRuler.change(100)
```

Comment:
Don't know if 100ms is the perfect throttle delay. Shorter would give us smoother movement, longer would reduce resource usage on the client. The ideal might be to have a long throttle but then use a native browser animation to do the movement, but it's probably not worth learning how to do that right now.